qml/Panel/Indicators/IndicatorsLight.qml

```diff
@@ -22,12 +23,140 @@ import Powerd 0.1
 import Lights 0.1
 import QMenuModel 0.1 as QMenuModel
 import Unity.Indicators 0.1 as Indicators
+import Wizard 0.1
```

Comment:
Why does it depend on Wizard?

Reply:
Oh i see for Powerd. We *should* move this to a common place at some point.

Reply:
I don't think the import of Wizard is needed any more. Whatever it was imported for doesn't seem to be a problem any more. I removed the import locally and it continues to function.